filter.go

```diff
@@ -32,6 +32,84 @@ type Filter struct {
 	} `json:"subscriptions"`
 }
 
+// FiltersList reflects a list of filters
+type FiltersList struct {
+	MaxResults int               `json:"maxResults" structs:"maxResults"`
+	StartAt    int               `json:"startAt" structs:"startAt"`
+	Total      int               `json:"total" structs:"total"`
+	IsLast     bool              `json:"isLast" structs:"isLast"`
+	Values     []FiltersListItem `json:"values" structs:"values"`
+}
+
+// FiltersListItem represents a Filter of FiltersList in Jira
+type FiltersListItem struct {
+	Self             string        `json:"self"`
+	ID               string        `json:"id"`
+	Name             string        `json:"name"`
+	Description      string        `json:"description"`
+	Owner            User          `json:"owner"`
+	Jql              string        `json:"jql"`
+	ViewURL          string        `json:"viewUrl"`
+	SearchURL        string        `json:"searchUrl"`
+	Favourite        bool          `json:"favourite"`
+	FavouritedCount  int           `json:"favouritedCount"`
+	SharePermissions []interface{} `json:"sharePermissions"`
+	Subscriptions    []struct {
```

Comment:
In the search result, `subscriptions` is the different data structure from the `Filter.subscriptions`.
So, I add this struct `FiltersListItem`.